next-cloudinary/src/components/CldVideoPlayer/CldVideoPlayer.tsx

```diff
@@ -131,8 +131,17 @@ const CldVideoPlayer = (props: CldVideoPlayerProps) => {
         }
       }
 
+      /* 
+      Parse the value passed to 'autoplay';
+      if its a boolean set it directly to browser standard prop autoplay else fallback to default; 
+      if its a string set it to cloudinary video player autoplayMode prop else fallback to default;
+      */
+      const autoPlayValue = typeof autoPlay === 'boolean'  ? autoPlay : false;
+      const autoplayModeValue = typeof autoPlay === 'string' ? autoPlay : 'never';
```

Comment:
do you think these two values should even define a default? just pass in undefined if not? that way it would default to the defaults that hte Player provides, in case for whatever reason that ever changes, or if the underlaying logic changes which handles the defined values

Reply:
Yes i do think they should define a default, the reason why i decided to fallback to their default was because i was of the opinion that setting the value to `undefined` would overide the defaults set by cloudinary.

Tho you are more experienced than me in using cloudinary, i can refactor the code if you believe setting the value to `undefined` doesn't disable or overide cloudinary defaults

Reply:
great point about `undefined` overwriting the values, that's been a gotcha before and i solved that by checking if the value is set before applying it as an option

that said, let's move forward with what you have and we can see how it goes. it's probably unlikely for the default to change, if ever, but if so, rarely, so we'll likely not run into any issues here (famous last words 😄 )